### PR TITLE
Benefit finder scenarios

### PR DESCRIPTION
**Pull request by nehemiah-abuga**
## PR Summary

This PR adds Cypress automated tests for QA Scenario 1, 2 and 3 that validates that correct benefit results are displayed based on query values in search parameter of URL.

The URL params for scenarios are document here https://docs.google.com/spreadsheets/d/1tCNJzKcQQvspkw2j30kkkv_XVFzghMXYEEd_SKSExq8/edit#gid=0
After you start storybook, you append the parameters  http://localhost:6006/iframe.html?args=&id=app--primary&viewMode=story& in storybook

## Related Github Issue

- fixes #(issue https://github.com/GSA/px-benefit-finder/issues/74)
The tickets under the above ticket are:
#651 
#652 
#653 

## Detailed Testing steps

- [ ]  `npm run cy:dev:storybook`
- [ ] Run ` npm run cy:run:e2e:chrome `
- [ ] Verify the test pass

### CODE REVIEW

File: benefit-finder/cypress/e2e/death-of-a-loved-one/selected-criteria-eligibility-benefits.cy.js [hunk 1]
```diff
@@ -98,4 +114,74 @@ describe('Validate correct eligibility benefits display based on selected criter
...
+    const benefitsCriteria = BENEFITS_ELIBILITY_DATA.scenario_2_veteran.en.param
+    const applicant_date_of_birth = encodeURI(
+      `{"month":"${
+        benefitsCriteria.applicant_date_of_birth_month - 1
+      }","day":"${benefitsCriteria.applicant_date_of_birth_day}","year":"${
+        benefitsCriteria.applicant_date_of_birth_year
+      }"}`
+    )
+    const deceased_date_of_death = encodeURI(
+      `{"month":"${benefitsCriteria.deceased_date_of_death_month - 1}","day":"${
```
Comment (scottqueen-bixal) [inline benefit-finder/cypress/e2e/death-of-a-loved-one/selected-criteria-eligibility-benefits.cy.js:156]: Please remove this and follow the pattern in scenario3

https://github.com/GSA/px-benefit-finder/pull/902/files#diff-b58f7ff9ab058d9ab02fc76409896fe396763c9930bf33ddc0e2040ad63bdcc3R174-R178

File: benefit-finder/cypress/e2e/death-of-a-loved-one/selected-criteria-eligibility-benefits.cy.js [hunk 2]
```diff
@@ -98,4 +114,74 @@ describe('Validate correct eligibility benefits display based on selected criter
         EN_DOLO_MOCK_DATA.data.benefits[22].benefit.eligibility[4].label
       )
   })
+
+  it('qa scenario 1 Covid EN - Verify correct benefit results for query values that includes covid in search parameter of URL', () => {
+    const benefitsCriteria = BENEFITS_ELIBILITY_DATA.scenario_1_covid.en.param
```
Comment (scottqueen-bixal) [inline benefit-finder/cypress/e2e/death-of-a-loved-one/selected-criteria-eligibility-benefits.cy.js:119]: Please remove this and follow the pattern in scenario3

https://github.com/GSA/px-benefit-finder/pull/902/files#diff-b58f7ff9ab058d9ab02fc76409896fe396763c9930bf33ddc0e2040ad63bdcc3R174-R178

File: benefit-finder/cypress/fixtures/benefits-eligibility.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                        "Survivors benefits for spouse"
+                    ]
+                },
+                "total_benefits": 30
+            }
+        }
+    },
+    "scenario_2_veteran": {
+        "en": {
+            "param": {
```
Comment (scottqueen-bixal) [inline benefit-finder/cypress/fixtures/benefits-eligibility.json:39]: Please remove this and follow the pattern in scenario3

https://github.com/GSA/px-benefit-finder/pull/902/files#diff-ebf47abf1eaef31294ffdbbbc744fb0a6bf48dc3bc09af80080b41b5880d33f7R84-R101

File: benefit-finder/cypress/fixtures/benefits-eligibility.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
+{
+    "scenario_1_covid": {
+        "en": {
+            "param": {
```
Comment (scottqueen-bixal) [inline benefit-finder/cypress/fixtures/benefits-eligibility.json:4]: Please remove this and follow the pattern in scenario3

https://github.com/GSA/px-benefit-finder/pull/902/files#diff-ebf47abf1eaef31294ffdbbbc744fb0a6bf48dc3bc09af80080b41b5880d33f7R84-R101

File: benefit-finder/package.json [hunk 5]
```diff
@@ -96,4 +96,4 @@
       "last 1 safari version"
     ]
   }
-}
+}
```
Comment (scottqueen-bixal) [inline benefit-finder/package.json:99]: please revert this change if unnecessary,

**Review by scottqueen-bixal — CHANGES_REQUESTED** [review]

File: benefit-finder/cypress/e2e/death-of-a-loved-one/selected-criteria-eligibility-benefits.cy.js (hunk 1 above)
Comment (nehemiah-abuga) [reply benefit-finder/cypress/e2e/death-of-a-loved-one/selected-criteria-eligibility-benefits.cy.js]: I have made the necessary updates

**Review by scottqueen-bixal — APPROVED** [review]
